package.json

```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/reflect/jason#readme",
   "license": "ISC",
-  "main": "index.js",
```

Comment:
I noticed this as well the other day... any clue how it could have worked before? 😕